apis/core/bootstrap/shapes/dataset.ts

```diff
@@ -30,7 +30,7 @@ const aboutQuery = sparql`construct {
   ?c ${hydra.member} ?about .
   ?about ${rdfs.label} ?name ; ?p ?o .
 } WHERE {
-  BIND ( iri('http://app.cube-creator.lndo.site/themes') as ?c )
+  BIND ( iri('http://app.cube-creator.lndo.site/about') as ?c )
```

Comment:
I mean, I know we talked about this on being slightly irrelevant but maybe let's actually use the env var here and nearby?

```suggestion
  BIND ( iri(`${env.API_CORE_BASE}about`) as ?c )
```

Reply:
Yeah, totally fine by me